src/components/Timeline.css

```diff
@@ -0,0 +1,43 @@
+.Timeline {
+  left: 15px;
+  position: absolute;
+  width: 3px;
+  margin: auto;
+  height: 500px;
+  background-color: #C4C4C4;
+}
+
+@media only screen and (min-width: 1215px) {
+  .Timeline {
+    left: 100px;
+  }
+}
+
+.TimelineDot {
+  z-index: 999;
+  position: absolute;
+  left: -11px;
+  height: 25px;
+  width: 25px;
+  color: #C4C4C4;
+  background-color: #C4C4C4;
+  box-shadow: 0 0 0 0.1em #C4C4C4 inset;
+  display: inline-block;
+  border-radius: 50%;
+}
+
+.TimelineDot > span {
+  z-index: 999;
+  position: absolute;
+  top: 3px;
+  left: 35px;
+  outline: none;
+}
+.TimelineDot > span:hover {
+  color: black;
+  cursor: pointer;
+}
+.TimelineDot.active {
+  color: black;
+  box-shadow: 0 0 0 0.3em #960F4A inset;
+}
```

Comment:
let's group timeline.jsx and timeline.css into a folder so we have components/timeline